website/pages/docs/advanced-topics/migrations.md

```diff
@@ -3,21 +3,16 @@
 This section describes how CloudQuery is dealing with schema changes in plugins.
 The overall idea is to not have breaking changes, but rather always add columns, because it is common for users to build views on top which we don't want to break. Those migration tactics are usually implemented in the destination plugins as source plugins are database agnostic and just send back JSON objects.
 
-## Addition
+CloudQuery has [two modes](/docs/reference/destination-spec#migrate_mode) of migrating to a new schema, `safe` which is supported by all destinations, and `forced` which is only supported by ClickHouse, MySQL, PostgreSQL, and SQLite at the moment.
 
-Column addition is the easy case where a source plugin adds additional column the destination plugin will not drop old one and will add a new one.
+The `safe` mode is the default and will not run migrations that would result in data loss, and will print an error instead. The `forced` mode will run migrations that may result in data loss and the migration should always succeed without errors.
 
-## Removal
-
-If a source plugin removes a column the destination plugin will not drop it and it will be up to the user to drop it if necessary or to do any other logic.
-
-## Rename
-
-Rename is basically Removal + Addition, meaning the destination plugin will just add additional column and new data will be saved there. It will be up to the user to do any migrations if needed.
-
-CloudQuery is an open-source cloud asset inventory powered by SQL, and as such, when plugins change their schema (change/remove columns) some migrations from the previous run are required. CloudQuery automatically drops and recreates those tables automatically as needed.
-
-## Type Change
-
-This is the most complex situation most probably and as a source plugin developer you want to avoid it if possible. On the destination plugin side, the column will be dropped an re-added.
+The following table describes when each mode is used:
 
+| Existing table schema change | Migration mode needed | Reasoning |
```

Comment:
I don't think "Migration mode needed" is clear, maybe re-arrange the table so that only forced migration requirement is left (remove first two rows) and add those as a footnote-ish paragraph telling both migrations can handle that as there are no changes needed.